src/supertux/menu/integrations_menu.cpp

```diff
@@ -41,12 +41,13 @@ enum IntegrationsMenuIDs {
 #ifdef ENABLE_DISCORD
   MNID_ENABLE_DISCORD,
 #endif
-  MNID_LEVELNAMES_EDITOR
+  MNID_LEVELNAMES_EDITOR,
+  MNID_LOCAL_FILES
 };
 
 IntegrationsMenu::IntegrationsMenu()
 {
-  add_label(_("Integrations"));
+  add_label(_("Social Integrations"));
   add_hl();
```

Comment:
Please separate integrations from add-on management, as they do not share any common purpose